src/NuGet.Clients/NuGet.VisualStudio/Extensibility/IVsPackageInstaller.cs

```diff
@@ -77,6 +79,7 @@ public interface IVsPackageInstaller
         /// A boolean indicating if assembly references from the package should be
         /// skipped.
         /// </param>
+        [Obsolete]
```

Comment:
consider adding a message, and explicitly stating `error: false` to not break consumers

Reply:
also, consider leaving this out to make this a pure doc change.  if you want to obsolete things, consider another PR for that?

Reply:
This API hasn't been supported for at least 7 years: https://github.com/NuGet/NuGet.Client/blame/c1a0597e5498bcf3bf0938b3dd0170edbf0b2d20/src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPackageInstaller.cs#L196-L200

Reply:
oh ok :)  no problem then.  #resolved.